src/coins.h

```diff
@@ -121,21 +121,6 @@ struct CCoinsCacheEntry
     CoinsCachePair* m_next{nullptr};
     uint8_t m_flags{0};
 
-    //! Adding a flag requires a reference to the sentinel of the flagged pair linked list.
-    static void AddFlags(uint8_t flags, CoinsCachePair& pair, CoinsCachePair& sentinel) noexcept
-    {
-        Assume(flags & (DIRTY | FRESH));
```

Comment:
we might want to mention in the commit message why we can safely remove this now

Reply:
Ok, agree, please resolve the comment